application/config/third_party.php

```diff
@@ -416,8 +416,9 @@
             'leaflet.css'
         ),
     ),
+    /* For geoname search autocomplete without jquery */
     'devbridge-autocomplete' => array(
-        'basePath' => 'third_party.devbridge-autocomplete.dist', /* For geoname search autocomplete without jquery */
```

Comment:
Was this intended to be removed on this PR?

Reply:
No … but it was not removed, just move comments. I fix it :)